user-guide/aragon-court/aragon-court-glossary.html

```diff
@@ -0,0 +1,124 @@
+	</strong>
+</p>
+<p>
+	<strong>Appeal</strong>
+</p>
+<p>
+	When someone disagrees with the preliminary ruling of a dispute, they can challenge the outcome and escalate the dispute to the next round of jury voting. Each appeal round brings in a larger and larger group of jurors until the fourth and final appeal round, when all active jurors are drafted to vote on the final ruling.
+</p>
+<p>
+	<strong>Aragon Court</strong>
+</p>
+<p>
+	A blockchain-based dispute resolution protocol that uses randomly selected jurors to resolve subjective disputes.
+</p>
+<p>
+	<strong>Aragon Network Juror (ANJ)<br>
+	</strong>
+</p>
+<p>
+	The native token of Aragon Court, used to incentivize coherent rulings and discourage attacks against the protocol.
+</p>
+<p>
+	<strong>Arguments<br>
+	</strong>
+</p>
+<p>
+	Text, audio, video, and images put forth to make a case in favor of one of the possible dispute outcomes (Allow, Block, or Refuse to vote).
+</p>
+<p>
+	<strong>Dispute<br>
+	</strong>
+</p>
+<p>
+	When an Aragon Court subscriber believes that an Agreement they are party to has been violated and they decide to try to block the disputable action or proposal using Aragon Court.
+</p>
+<p>
+	<strong>Draft<br>
+	</strong>
+</p>
+<p>
+	The process of randomly selecting jurors for a new dispute round from the pool of active jurors.
+</p>
+<p>
+	<strong>Final ruling<br>
+	</strong>
+</p>
+<p>
+	<strong></strong>The final ruling of a dispute is one of either:
+</p>
+<ol>
+	<li>The ruling that a plurality of jurors in the current round have voted for, provided there has been no appeal or it is the final appeal round, or</li>
+	<li>The ruling proposed by an appealing party, provided that no one challenges the appeal.</li>
+</ol>
+<p>
+	<strong>Juror<br>
+	</strong>
+</p>
+<p>
+	A participant in the Aragon Court protocol who has locked and activated a specified minimum amount of ANJ, making them eligible to be drafted to resolve disputes brought before the court.
+</p>
+<p>
+	<strong>Period</strong>
+</p>
+<p>
+	Disputes are broken up into multiple periods, which are themselves each made up of a specific number of terms. The periods of a dispute are:
+</p>
+<ul>
+	<li>Argument submission (optional)</li>
+	<li>Vote commit</li>
+	<li>Vote reveal</li>
+	<li>Appeal</li>
+	<li>Appeal challenge</li>
+</ul>
+<p>
+	<strong>Preliminary ruling<br>
+	</strong>
+</p>
+<p>
+	The ruling selected by the plurality of jurors in the current dispute round, which may become the final ruling provided there is no appeal.
+</p>
+<p>
+	<strong>Rewards<br>
+	</strong>
+</p>
+<p>
+	Aragon Court offers financial rewards to users who successfully perform certain actions, such as drafting jurors, appealing preliminary rulings, and voting in favor of the final ruling of a dispute. Some rewards, such as dispute fees, ruling fees, and subscription fees, are only given to active jurors. Other rewards, such as appeal fees and maintenance fees, can be earned by anyone.
```

Comment:
Not sure what the "maintenance fee" are referring to (I may be out of the loop).

Reply:
Yes, there is another "maintenance fee" for the settlements. After jurors have ruled a dispute, we need to slash the losing jurors and reward the winning ones, and the same for the appeal parties if there is any.